lib/improver/tests/weights/test_ChooseDefaultWeightsNonLinear.py

```diff
@@ -33,74 +33,59 @@
 
 import unittest
 
-from cf_units import Unit
```

Comment:
At the top underneath the license the doc string needs to be changed to say NonLinear.